auxiliary/producers/replayer/README.md

```diff
@@ -2,3 +2,13 @@ For debugging, use
 ```bash
 bazel run //auxiliary/producers/replayer -- --port 8080 --mode constant --log debug
```

Comment:
for the remote double tunneled MongoDB this is the correct port?

Reply:
This is the service port, not the database port!